endpoint-sec/src/event.rs

```diff
@@ -409,6 +411,7 @@ macro_rules! make_event_data_iterator {
         impl<'raw> std::iter::Iterator for $name<'_, 'raw> {
             type Item = $item;
 
+            #[allow(unused_unsafe)]
```

Comment:
Rather than this, I wonder if we could "hoist" the `unsafe` up. Like, could we have two variants of this macro, one where we call it as

```rust
make_event_data_iterator!(
    OdMemberIdArray;
    /// Iterator over the uuids in an [`OdMemberIdArray`]
    OdMemberIdArrayUuids with member_count (usize);
    libc::uuid_t;
    read_nth_uuid,
    unsafe { std::convert::identity },
);
```

Reply:
I only use the `allow` for one thing, and it's fully internal, not a bother I think